test/main/transaction/index.test.js

```diff
@@ -485,3 +493,149 @@ describe('#sign', () => {
     })
   })
 })
+
+describe('#classifyTransaction', () => {
+  const method = 'eth_sendTransaction'
+  const from = '0xd8da6bf26964af9d7eed9e03e53415d37aa96045'
+  const to = '0x2f3a40a3db8a7e3d09b0adfefbce4f6f81927557'
+  const Request = (param) => ({
+    method,
+    params: [param]
+  })
+
+  // Contract deployments
+  it('should correctly classify standard contract deployments', () => {
```

Comment:
I think the test name should explain the result, so something like `should classify requests with no recipient as contract deployments`

Reply:
in general test names with "should do X correctly" aren't as useful because:

1. theyre really just re-stating what the test is doing
2. its not clear _why_ that would be correct or incorrect